### PR TITLE
PPF-002: Create the landing page

### PR DESCRIPTION
**Pull request by Redturtle42**

### CODE REVIEW

File: src/footer/footer.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.footer {
+  background-color: #90fa69;
```
Comment (1amG4bor) [inline src/footer/footer.scss:2]: Could you extract the colors into variables in a common .scss file and use it everywhere to keep the colors consistent throughout the application?
_The mentioned common .scss file might contain common style classes as well._

File: src/footer/footer.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Col, Container, Row } from "react-bootstrap";
+import "./footer.scss";
+
+export const Footer = () => {
+  return (
+    <Container fluid className="text-center footer">
+      <Row>
+        <Col>
+          <svg
```
Comment (1amG4bor) [inline src/footer/footer.tsx:10]: The code would be cleaner if these SVG came from the `svg.tsx` like on the home page..
What do you think about it?

File: src/home/home.scss [hunk 3]
```diff
@@ -0,0 +1,99 @@
+.home {
+  height: 100%;
+  width: 100%;
+  font-family: "Pacifico", cursive;
+  font-size: 1.5rem;
+
+  h1 {
```
Comment (1amG4bor) [inline src/home/home.scss:7]: I assume this style is overridden with the upcoming h1 section in line 30.

File: src/footer/footer.tsx (hunk 2 above)
Comment (Redturtle42) [reply src/footer/footer.tsx]: Moved the svg-s into the svg.tsx file.

File: src/footer/footer.scss (hunk 1 above)
Comment (Redturtle42) [reply src/footer/footer.scss]: Done.

**Review by 1amG4bor — APPROVED** [review]

**Pull request merged**